db/migrate/20170602111120_add_timestamps_to_documents.rb

```diff
@@ -0,0 +1,110 @@
+class AddTimestampsToDocuments < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def change
+    add_timestamps :documents, null: true
+    puts "> Timestamp fields added."
+    puts "> Updating multiple-locale documents without editions..."
+
+    update_non_en_documents_without_editions
+    puts "> Multiple-locale documents without editions updated."
+    puts "> Updating remaining documents..."
+
+    Document.where(created_at: nil).find_each do |document|
+      editions = Edition.where(document_id: document.id).order(:user_facing_version)
+      if editions.present?
```

Comment:
if you change this to `.exists?` it might run a bit faster - `present?` http://www.ombulabs.com/blog/benchmark/performance/rails/present-vs-any-vs-exists.html